.travis.yml

```diff
@@ -38,20 +24,11 @@ jobs:
     - stage: test
       env:
         - TESTS="1" ODOO_REPO="OCA/OCB" MAKEPOT="1"
-    # Other env possibilities:
-    # use this to install a standalone database to export .pot files
-    # - MAKEPOT="1"
-    # either use the two lines above or the two below. Don't change the default if
-    # it's not necessary (it is only necessary if modules in your repository can't
-    # be installed in the same database. And you get a huge speed penalty in your
-    # tests)
-    #  - TESTS="1.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1"
-    #  - TESTS="1.0" ODOO_REPO="OCA/OCB" UNIT_TEST="1"
 
 env:
   global:
     - VERSION="8.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
-    - WKHTMLTOPDF_VERSION="0.12.1"
+    - WKHTMLTOPDF_VERSION="0.12.5"
```

Comment:
I thought that each Odoo version had its matching wkhtml version, ok for that

Reply:
Yes, but I've no better solution to offer and I guess for running tests it should be good enough.